sources/views/contactsData/contactsTemplate.js

```diff
@@ -74,4 +122,87 @@ export default class ContactsTemplateView extends JetView {
 			this.contactsTemplate.setValues(contact);
 		});
 	}
+
+	editContact() {
+		this.show("./contactsData.contactForm");
+	}
+
+	deleteContact() {
+		webix.confirm({
+			text: "Contact's info and all related activities will be deleted permanently! Continue?"
+		}).then(() => {
+			const contactId = this.getParam("id", true);
+			let actsToDelete;
+			let filesToDelete;
+			if (this.isNumber(contactId)) {
+				actsToDelete = activities.find(act => +act.ContactID === contactId);
```

Comment:
Here and everywhere else compare IDs as strings.
- in theory, backend won't always give you numeric IDs, so string comparison is more consistent
- Webix API may convert some values to strings - keep that in mind as well

Reply:
fixed